src/flux/media.coffee

```diff
@@ -2,6 +2,26 @@ _ = require 'underscore'
 htmlparser = require 'htmlparser2'
 {makeSimpleStore} = require './helpers'
 
+LINKS_BEGIN = ['#']
+LINKS_CONTAIN = ['/contents/', '.cnx.org']
```

Comment:
Also, what happens if the link is `http://cnx.org/contents/[UUID]`?

- this returns blank HTML that contains a `<script>` tag to load webview
- Not sure if browsers will like `http:` ajax requests from a `https:` site